.gitlab/ci/on-push.yml

```diff
@@ -26,6 +26,8 @@ trigger-private-build:
 run-unittests-and-lint:
   extends:
     - .run-unittests-and-lint
+  rules:
+    - if: '$BUCKET_UPLOAD == "false"'
```

Comment:
isn't the default value is empty (variable doesn't exist)?
looking at your last build, this job did not run in your regular flow.

Reply:
correct
changed it.
https://code.pan.run/xsoar/content/-/pipelines/2236902